contrib_bots/bot_lib.py

```diff
@@ -33,12 +35,18 @@ def is_legal(self):
         else:
             return True
 
+    def show_error_and_exit(self):
+        logging.error(self.error_message)
+        sys.exit(1)
```

Comment:
Agreed! Maybe we can open an issue for this, since the momentary code uses exit() as well.